pkg/network/bridge/plugin.go

```diff
@@ -299,6 +301,8 @@ func attachPodToBridge(bridgeName, nicName string, containerPid int) error {
 		return err
 	}
 
+	nbdp.interfaceCleaner.Register(nicName, containerPid)
```

Comment:
you should first wait for the cleaner thread to start. though it is unlikely, it may be that the cleaner did not start yet

Reply:
Done

Reply:
:+1: